src/lib.rs

```diff
@@ -907,6 +955,13 @@ impl Tool {
     }
 }
 
+fn get_mtime<P: AsRef<Path>>(file: P) -> u64 {
+    use filetime::FileTime;
+    fs::metadata(file)
+        .map(|mtime| FileTime::from_last_modification_time(&mtime).seconds())
```

Comment:
Sometimes the nanosecond precision here might matter, perhaps that could be preserved?